custom_from.php

```diff
@@ -134,10 +264,88 @@ public function message_compose($params)
                         $score = 1;
                     }
                 }
-            }
+
+                if ($score > 2 && !$should_override_identity) {
+                    break;
+                }
+
+                /** Remove autocomplete by JS frontend after page loaded */
+                $rcmail->output->set_env('identities', array());
+
+                break;
+
+            case self::DEFAULT_IDENTITY_OPTION:
+                $default_identity = $rcmail->user->get_identity();
+                $select_from_attrib = array_intersect_key($attrib, array_flip(array('id', 'class', 'style', 'size', 'tabindex'))) + array(
```

Comment:
I don't understand this part that seems to be trying to extract HTML attributes out of `$attrib`. `tabindex` does exist but none of the other. What is the intent here?